### PR TITLE
[Fix] Active Directory group lookup timing out

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [x] Ready

## Related Issues
related: https://github.com/demisto/etc/issues/26114

## Description
Added `time_limit` argument to `ad-get-group-members` command. Default is 30 seconds.

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by DeanArbel** [comment]
@kirbles19 Please review the following files:
[Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.yml](https://github.com/demisto/content/blob/eb5195de61c3978ae77181ebbd292967b2a15954/Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.yml)
[Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md](https://github.com/demisto/content/blob/eb5195de61c3978ae77181ebbd292967b2a15954/Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md)
[Packs/Active_Directory_Query/ReleaseNotes/1_0_2.md](https://github.com/demisto/content/blob/eb5195de61c3978ae77181ebbd292967b2a15954/Packs/Active_Directory_Query/ReleaseNotes/1_0_2.md)

File: Packs/Active_Directory_Query/ReleaseNotes/1_0_2.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Active Directory Query v2
+- Added `time_limit` argument to `ad-get-group-members` command. Default is 30 seconds.
```
Comment (yuvalbenshalom) [inline Packs/Active_Directory_Query/ReleaseNotes/1_0_2.md:4]:
```suggestion
- Added `time_limit` argument to `ad-get-group-members` command. Default is 180 seconds.
```

File: Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py [hunk 2]
```diff
@@ -463,6 +463,7 @@ def search_group_members(default_base_dn, page_size):
     args = demisto.args()
     member_type = args.get('member-type')
     group_dn = args.get('group-dn')
+    time_limit = int(args.get('time_limit', 30))
```
Comment (yuvalbenshalom) [inline Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py:466]:
```suggestion
    time_limit = int(args.get('time_limit', 180))
```
lets make the the defaults the same

**Comment by kirbles19** [comment]
@DeanArbel Done.

**Pull request merged**